lib/jwt_sessions/store_adapters/redis_store_adapter.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+module JWTSessions
+  module StoreAdapters
+    class RedisStoreAdapter < AbstractStoreAdapter
+      attr_reader :redis_client
+
+      def initialize(token_prefix: JWTSessions.token_prefix, **options)
```

Comment:
Looks like this implementation won't pick up settings specified as follows
```
JWTSessions.redis_port = 6378
JWTSessions.token_store = :redis
```

Reply:
https://github.com/tuwukee/jwt_sessions/pull/39/files#diff-f9f1ac5ee448bd383409144652d60a5dR93 is responsible for that - it takes all the passed args and after that makes a `||=` fallback to the module-level config options. I've added a test to verify that